### PR TITLE
1st task of xmljson

### PR DESCRIPTION
**Pull request by IvayloTsvetkov**
Parsing simple class to xml and json using Xstream and Gson.

### CODE REVIEW

File: xmljson_1/src/test/java/com/clouway/JSONCodecTest.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+import org.junit.Test;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.junit.Assert.*;
+
+public class JSONCodecTest {
+  JSONCodec jsonCodec = new JSONCodec();
+
+  @Test
+  public void personEqualDeserializedJsonPersonTest() {
```
Comment (mgenov) [inline xmljson_1/src/test/java/com/clouway/JSONCodecTest.java]: decodeEncodedPerson()

File: xmljson_1/src/test/java/com/clouway/XMLCodecTest.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+import org.junit.Test;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.junit.Assert.*;
+
+public class XMLCodecTest {
+  XMLCodec xmlCodec = new XMLCodec();
+
+  @Test
+  public void personEqualDeserializedXmlPersonTest() {
```
Comment (mgenov) [inline xmljson_1/src/test/java/com/clouway/XMLCodecTest.java]: decodeEncodedPerson

File: xmljson_1/src/main/java/com/clouway/Main.java [hunk 3]
```diff
@@ -0,0 +1,19 @@
+package com.clouway;
+
+public class Main {
+  public static void main(String[] args) {
+
+    MessageCodec xmlCodec = new XMLCodec();
+    Person person = new Person("Ivaylo", "Tsvetkov");
+    String personToXML = xmlCodec.serialize(person);
```
Comment (mgenov) [inline xmljson_1/src/main/java/com/clouway/Main.java]: String xmlContent = xmlCodec.serialize(person);

**Comment by IvayloTsvetkov** [comment]
Applied the requested name changes to both tasks.

**Pull request merged**